index.md

```diff
@@ -73,7 +75,7 @@ I use this method along with…
 
 #### Asana
 
-I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) around a year ago, and have been using it on and off for long-term planning of my activities.
+I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) and have been using it on and off for long-term planning of my activities.
 As a PhD student, you will often need to plan well into the future, with many external deadlines looming ahead: progress reports, vivas, conference submissions etc.
```

Comment:
More related to the sections on text formats and markdown: how do you deal with the back-and-forth of working e.g. with .md and Atom, but then having to send advisors .docx and then incorporating their changes/comments back into your original file(s)? Some advice on that would be useful. I see you have one or two sentences on this; care to expand? ("I export the .md files to .odt/.doc(x), send them for comments to my supervisors, and incorporate the suggestions/edits back into the .md file.")